src/test/scala/format/pgn/DumperTest.scala

```diff
@@ -66,6 +67,18 @@ class DumperTest extends ChessTest {
     E2 -> A6
   )
 
+  val threeCheck = Game(Board init ThreeCheck).playMoves(
```

Comment:
Somewhat arbitrary three check game that gets the checks in in as few moves as I could figure out :laughing:.